### PR TITLE
Fix various code format, style issues

### PR DESCRIPTION
**Pull request by birkholz**
* PEP8
* Fix shadowed var names
* Enforce text `id` in JWTs
* Remove unused imports/code

### CODE REVIEW

File: zc_common/jwt_auth/utils.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
     
-    :param User: an object with `pk` and `get_roles()`
+    :param user: an object with `pk` and `get_roles()`
     :return: A dictionary that can be passed into `jwt_encode_handler`
-    '''
+    """
 
     payload = {
-        'id': user.pk,
+        'id': encoding.force_text(user.pk),
```
Comment (birkholz) [inline zc_common/jwt_auth/utils.py]: @ograycode This will enforce that the ID in JWTs is a string

File: zc_common/remote_resource/mixins.py [hunk 2]
```diff
@@ -1,24 +0,0 @@
-"""
```
Comment (ograycode) [inline zc_common/remote_resource/mixins.py]: Did you mean to delete this?

Comment (birkholz) [inline zc_common/remote_resource/mixins.py]: Yes. This was replaced with the generic `?filter[]` system in #25 . We just missed this.

**Comment by ograycode** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;fUe9EsdO&#x2F;606-cannot-pass-belongs-to-check-because-request-user-id-is-an-int"> Cannot pass `belongs_to` check because `request.user.id` is an int</a>

File: zc_common/remote_resource/mixins.py (hunk 2 above)
Comment (birkholz) [inline zc_common/remote_resource/mixins.py]: [Proof of it's non-use](https://github.com/search?q=org%3AZeroCater+MultipleIDMixin&type=Code)

**Pull request merged**